backend/dal/Services/PropertyService.cs

```diff
@@ -80,6 +81,73 @@ public Paged<Property> GetPage(PropertyFilter filter)
 
             return new Paged<Property>(items, filter.Page, filter.Quantity, query.Count());
         }
+
+        /// <summary>
+        /// Get the property for the specified primary key 'id' value.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        public Property Get(int id)
+        {
+            this.User.ThrowIfNotAllAuthorized(Permissions.PropertyView);
+
+            var property = this.Context.Properties
+                .Include(p => p.District)
```

Comment:
All the includes are bad for performance... At this point I don't think it matters.  We could improve performance on something like this by using a view, or flattening the schema.